h/views/main.py

```diff
@@ -66,9 +66,9 @@ def robots(_context, request):
     csp_insecure_optout=True,
 )
 def stream(_context, request):
-    q = request.params.get("q", "").split(":", 1)
-    if len(q) >= 2 and q[0] == "tag":
-        tag = q[1]
+    q_param = request.params.get("q", "").split(":", 1)
+    if len(q_param) >= 2 and q_param[0] == "tag":
+        tag = q_param[1]
```

Comment:
`q_param` seems literal rather than informative. What does the `q` param represent. Is it a `query` or `terms` or something in this context?